packages/client/src/utils/enrichDataBinding.js

```diff
@@ -0,0 +1,46 @@
+import mustache from "mustache"
```

Comment:
Running this with handlebars instead of mustache may be beneficial in the future, much more expanded syntax which could be useful (and is drop in).

Reply:
Good point! Would be sweet to enable other features. Will drop in handlebars instead.

Reply:
Handlebars doesn't seem to work as nicely as mustache here actually - it doesn't like spaces in variable names which we currently have lots of. To work with spaces you need to use the literal reference syntax like `{{[foo bar]/expression}}` which would require lots of other changes to our binding syntax. I think I'll leave in mustache until we need more powerful features and we can revisit this since it would likely require other changes anyway.